From 60e386048bba4cc54a6a9d8338f5ebc76c1e18e6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Wed, 11 Sep 2019 22:00:39 +0200
Subject: [PATCH 067/152] backgroundMenu: Animate menu opening

The animation was removed in commit 6a00a504d4fc for consistency with
other menus. However commit a9b12d5d737dc then *added* animations to
those just four minutes later.

So add back the original animations for consistency, both with menu
closing and with other menus.

https://gitlab.gnome.org/GNOME/gnome-shell/issues/1595
---
 js/ui/backgroundMenu.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/backgroundMenu.js b/js/ui/backgroundMenu.js
index b19ced9e4..ec39d1290 100644
--- a/js/ui/backgroundMenu.js
+++ b/js/ui/backgroundMenu.js
@@ -31,7 +31,7 @@ function addBackgroundMenu(actor, layoutManager) {
 
     function openMenu(x, y) {
         Main.layoutManager.setDummyCursorGeometry(x, y, 0, 0);
-        actor._backgroundMenu.open(BoxPointer.PopupAnimation.NONE);
+        actor._backgroundMenu.open(BoxPointer.PopupAnimation.FULL);
     }
 
     let clickAction = new Clutter.ClickAction();
-- 
2.23.0

